package.json

```diff
@@ -17,6 +17,7 @@
     "antd": "3.20.4",
     "array-pack-2d": "^0.1.1",
     "array-range": "^1.0.1",
+    "bn.js": "^5.1.3",
```

Comment:
Let's add bn.js as a peer dependency rather a strict one so we don't end up getting even more bloat. And let's make it super super permissive (i think >= 4.0.0)

```
"peerDependencies": {
    "bn.js": ">= 4.0.0"
  }
```

web3 itself uses ^4.11.9
https://github.com/ethereum/web3.js/blob/1.x/package.json#L104

Reply:
good idea